build

```diff
@@ -51,6 +61,8 @@ function main() {
     rm -f $SRCDIR/lib/puppy
     rm -f $SRCDIR/lib/dogstatsd
     rm -f $SRCDIR/lib/trace-agent
+    # rm -f $SRCDIR/lib/agent-binaries.zip
```

Comment:
Let's remove this if it is not needed

Reply:
I meant to make a comment on this file. We need all the commented out lines, I just don't have a link to this agent artifact yet. Once I do we'll need to use it.

Reply:
Ok, these links are updated now !!!